src/components/atoms/TxBarbleIcon.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    class="icon icon-tabler icon-tabler-barbell-off"
```

Comment:
Is these classes needed?

Reply:
I guess for now the class is unnecessary since there is no style bound to it.
the class above is considered a default class from tabler icons: https://tabler-icons.io/
but I think I might remove the class and change the component name because the class name that is used in the component above is also the default name for the icon

Reply:
Is there any reason why you didn't use SVG formats? Actually, putting the icons into /src/assets/icon/*.svg is also okay instead of make it as component

Reply:
svg is just a common image formats, you can always use it using `<img>` tag

```
<img src="@/assets/icon/barbel.svg" />
```

Reply:
The reason I keep the SVG formats is because of the customization
I can directly apply styles like the `fill` color, `stroke-width`, etc to the SVG in that component

Reply:
I would do the same thing when the `fill` and `stroke-width` customizable via props

Reply:
but if it's static and won't be changed I'll stick with `*.svg` format

Reply:
But if you know what you're doing, then go on 👍

Reply:
okay, thank you, sir 
for now ill keep it that way, until I found the best method to do it

Reply:
![51fm5uHUuoL _AC_UF1000,1000_QL80_](https://user-images.githubusercontent.com/18502234/230583089-7134492b-3f1e-461c-b137-88e4b1b4f41e.jpg)